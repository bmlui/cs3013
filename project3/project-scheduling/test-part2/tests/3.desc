analysis: FIFO: with different arrival time
