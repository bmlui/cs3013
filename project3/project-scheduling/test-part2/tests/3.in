0,20
0,19
1,18
1,17
2,16
3,15
4,14