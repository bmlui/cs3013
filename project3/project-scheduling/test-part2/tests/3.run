./scheduler 1 FIFO tests/3.in 0
