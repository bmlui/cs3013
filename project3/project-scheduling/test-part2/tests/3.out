Execution trace with FIFO:
t=0: [Job 0] arrived at [0], ran for: [20]
t=20: [Job 1] arrived at [0], ran for: [19]
t=39: [Job 2] arrived at [1], ran for: [18]
t=57: [Job 3] arrived at [1], ran for: [17]
t=74: [Job 4] arrived at [2], ran for: [16]
t=90: [Job 5] arrived at [3], ran for: [15]
t=105: [Job 6] arrived at [4], ran for: [14]
End of execution with FIFO.
Begin analyzing FIFO:
Job 0 -- Response time: 0  Turnaround: 20  Wait: 0
Job 1 -- Response time: 20  Turnaround: 39  Wait: 20
Job 2 -- Response time: 38  Turnaround: 56  Wait: 38
Job 3 -- Response time: 56  Turnaround: 73  Wait: 56
Job 4 -- Response time: 72  Turnaround: 88  Wait: 72
Job 5 -- Response time: 87  Turnaround: 102  Wait: 87
Job 6 -- Response time: 101  Turnaround: 115  Wait: 101
Average -- Response: 53.43  Turnaround 70.43  Wait 53.43
End analyzing FIFO.
